flow.js

```diff
@@ -93,6 +95,12 @@ class Flow {
       'repair-living-areas-lighting-problems': { prevStep: 'repair-living-areas-problems', nextStep: 'repair-description'},
       'wall-floor-ceiling-problems': {nextStep: 'repair-description'},
       'repair-kitchen-cupboard-problems': {prevStep: 'repair-kitchen-problems', nextStep: 'repair-description'},
+      'bathroom-damp-mould-problems': { prevStep: 'repair-bedroom-problems',
```

Comment:
I believe the `prevStep` should be `repair-bathroom-problems` instead of `repair-bedroom-problems`.
```suggestion
      'bathroom-damp-mould-problems': { prevStep: 'repair-bathroom-problems',
```